app/move/fields/create.js

```diff
@@ -253,31 +262,24 @@ module.exports = {
   // move details
   to_location: {},
   move_type: {
+    ...cloneDeep(moveType),
+    items: [{ ...items.moveTypeCourtAppearance }],
+  },
+  move_type__oca: {
```

Comment:
Can we remove the associating from to the role from this name. This journey may not always be tied to that specific role.

I know it's difficult to not use the term OCA within the code because that's how we've been referring to this journey in standups, tickets, etc but I think we need to ensure that we keep the role out of logic and naming of things at this level.

Reply:
Fair enough, but how do we call it then?